core/src/main/java/feign/Request.java

```diff
@@ -190,17 +190,6 @@ public Charset charset() {
     return body.encoding;
   }
 
-  /**
-   * If present, this is the replayable body to send to the server. In some cases, this may be
-   * interpretable as text.
-   *
-   * @see #charset()
-   * @deprecated use {@link #requestBody()} instead
-   */
-  public byte[] body() {
```

Comment:
I don't recall when we deprecated this.

If was on 9.x think is ok to drop it.

Reply:
If deprecated method is completely removed from usage, is not that ok to remove the method itself? I think removing it prevents further usage. Should I keep it or remove it?

Reply:
A `decoder` may exists out there that still uses it.

Reply:
I see.. Sorry I am not fully familiar on source code. I'll put it back.

Just a question out of this context, is it acceptable to use a 3rd party library (like apache commons lang) for some requirement?

Reply:
core has being clean from external dependencies and we are most likely to want to keep it that way.

Particularly about apache commons-*, every time I see them I read "old deprecated code that somehow still survived" =)